app/app.service.ts

```diff
@@ -1,8 +1,6 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
-import { Observable } from 'rxjs/Observable';
-import 'rxjs/add/operator/map';
-import 'rxjs/add/operator/catch';
+import { Observable } from 'rxjs';
 
```

Comment:
Why you changed those? is everything still working and tested in demo app after this?

Reply:
With RxJS 6, we now need to import operators only in files that need it, here is a good example: https://github.com/ng2-ui/auto-complete/pull/351/files#diff-8aaf19bd0c967a0eaa473a82229fe2d8R4